src/main/java/io/nats/client/impl/PullStatusMessageManager.java

```diff
@@ -23,22 +23,13 @@ class PullStatusMessageManager extends MessageManager {
 
     private static final List<Integer> PULL_KNOWN_STATUS_CODES = Arrays.asList(404, 408, 409);
 
-    private int lastStatusCode = -1;
```

Comment:
this field an associated getter were not used as noted by the IDE. So removed them.